qutebrowser/commands/runners.py

```diff
@@ -80,21 +80,23 @@ def __init__(self, win_id, partial_match=False, parent=None):
         self._partial_match = partial_match
         self._win_id = win_id
 
-    def _get_alias(self, text):
+    def _get_alias(self, text, default=None):
         """Get an alias from the config.
 
         Args:
             text: The text to parse.
+            default : Default value to return when alias was not found. By
+                      default it is set to None.
```

Comment:
nitpick: No point in stating this here as there's already `default=None` above :wink: I'll fix it up before merging unless other things come up while reviewing.